### ISSUE TITLE
Add user-secrets "get" command to CLI

### ISSUE THREAD

**Issue by TopperKain** [dotnet/aspnetcore#27045]
### Is your feature request related to a problem? Please describe.
I am trying to utilize user secrets in a CLI pipeline, but I cannot easily access a named secret via the command line.

My specific scenario is automating the ef core scaffolding of a database via the command line. I'd like to store my DB connection string in the development user secrets repository, set it to a variable, and then use it in the ef core scaffold command, but I'm sure there other places for developer tooling to utilize access to the user-secret store on the command line.

### Describe the solution you'd like
I would like the dotnet user-secrets tooling to support a " get " command to retrieve a secret value by id. This would be invoked via 
``` PowerShell
dotnet user-secrets get myKey
```
And would return the string for the value associated with myKey.

Example usage 1:
``` PowerShell
dotnet user-secrets set myKey "cool value" # returns "Successfully saved myKey = cool value to the secret store."
dotnet user-secrets get myKey # returns "cool value"
```

Example usage 2:
``` PowerShell
dotnet user-secrets set myConnString "User Id=Sample; Password=****" # returns "Successfully saved myKey = cool value to the secret store."
$myConnString = dotnet user-secrets get myConnString 
dotnet ef scaffold $myConnString Microsoft.EntityFrameworkCore.SqlServer # Scaffolds the database defined in myConnString
```

Alternatives considered:
I attempted to do this via the existing "list" command, but there is no way to get an individual key/value pair without utilizing a form of "grep" or transforming to JSON and back, which is unwieldly and user hostile.

I have also utilized alternative secret storage (such as Azure Key Vault), but this is not a 1-1 change of experience and can involve some unwieldy set up as well.

**Comment by jikuja** [comment]
Indeed. Getting keyvault /web app configuration/app configuration service compatible results from `dotnet user-secrets list` would be a cool feature.

**Comment by shirhatti** [comment]
Triage: this isn't what we consider idiomatic usage of dotnet user-secrets. In environments like CI, you're better off using the environment variable configuration source. As such, I'm closing this issue for now.

We will re-consider if there is sufficient demand for this feature

**Comment by LiteracyFanatic** [comment]
I would really like to see this feature implemented. During development I usually have multiple terminal tabs open to different directories in a project for various tasks (one running an aspnet server, one running a webpack server, one with an ssh connection to a remote box, and one for running commands like git or ef migrations). It's annoying to manually open and configure each of these terminals, so I have a script which opens them, sets them to the correct directories, and sets appropriate environment variables. Having a `get` command would allow me to set connection strings from my script without having to hard code the values or the location they are stored in. It's not hard to parse the `list` output or just pipe the contents of the JSON file through `jq` to grab what I need, but according to the documentation we shouldn't need to know the implementation details to make use of the `user-secrets` tool.
